src/main/java/com/indeed/jiraactions/ActionFactory.java

```diff
@@ -32,9 +32,10 @@ public ActionFactory(final UserLookupService userLookupService,
     public Action create(final Issue issue) throws IOException {
         final User assignee = userLookupService.getUser(issue.initialValueKey("assignee", "assigneekey"));
         final User reporter = userLookupService.getUser(issue.initialValueKey("reporter", "reporterkey"));
+        final User actor = userLookupService.getUser(issue.fields.creator == null ? User.INVALID_USER.getKey() : issue.fields.creator.getKey());
```

Comment:
Haven't we already done the userLookup for the actor? Action.actor() returns an object of type `User`.